addons/l10n_in/data/template/account.account-in.csv

```diff
@@ -12,10 +12,10 @@
 "p10071","Prepaid Insurance","10071","asset_current","","False"
 "p1011","Buildings","1011","asset_fixed","","False"
 "p1012","Land","1012","asset_fixed","","False"
-"p1013","Equipment","1013","asset_fixed","","False"
-"p1014","Vehicle","1014","asset_fixed","","False"
-"p1015","Computer/Laptops (Assets)","1015","asset_fixed","","False"
-"p1016","Furniture","1016","asset_fixed","","False"
+"p1013","Equipment","1013","asset_fixed","l10n_in.account_tag_property_plant_and_equipment","False"
```

Comment:
it seems to me that this is not necessary as we check the internal group in the balance sheet report. I think we should check specific accounts that are not covered based on the Account Coverage report and then see why from there

Reply:
any update on this? :slightly_smiling_face: also, the style check is failing